h2o-r/tests/testdir_jira/runit_pubdev_6488.R

```diff
@@ -0,0 +1,137 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+
+
+test.tree.fetch <- function() {
+  
+  data <- as.h2o(data.frame(
+    x1 = c(1, NA, 2),
+    x2 = c(2, NA, 1),
+    y = c(1, 2,3)
+  ))
+  
+  # A simple model to test prediction the correctly printed by delegating to print.H2ONode.
+  model <- h2o.xgboost(x= c("x1", "x2"), y = "y", training_frame = data, max_depth = 0, ntrees = 1)
+  tree <- h2o.getModelTree(model, 1)
+  expect_false(is.null(tree))
+  expect_equal(3, grep("Prediction is 0", capture.output(print(tree@root_node))))
+  
+  # Random Tree API test with a randomly generated matrix
+  nrows <- 1000
+  ntrees <- 50
+  response <- sample(0:1,nrows,replace = TRUE)
+  domain <- unique(response)
+  data <- data.frame(
+    x1 = rnorm(nrows),
+    x2 = rbinom(nrows, 1, 0.5),
+    x3 = rbinom(nrows, 10, 0.5),
+    response = response # Binomial response
+  )
+  
+  data <- as.h2o(data)
```

Comment:
Do we expect for some random data set it will fail? If yes, I would add seed to be able to reproduce a failure.

Reply:
Random printed seed could be a way.